test/unit/lib/configuration/variables/sources/instance-dependent/get-sls.test.js

```diff
@@ -13,18 +13,24 @@ describe('test/unit/lib/configuration/variables/sources/instance-dependent/get-s
   let variablesMeta;
   let serverlessInstance;
 
-  before(async () => {
+  const initializeServerless = async (providerStage, cliOptionStage) => {
```

Comment:
Let's update the signature to `(configExt, options)`. It'll be more up to convention in other test files, and more portable for eventual future addition.

`configExt` should be a simply configuration extension to be applied via `_.merge(config, configExt)`, and `options`,  should be CLI options.